packages/server/src/tests/banners/bannerSelection.ts

```diff
@@ -101,6 +100,7 @@ export const selectBannerTest = async (
     pageTracking: PageTracking,
     baseUrl: string,
     getTests: () => Promise<BannerTest[]>,
+    targetingTests: TargetingTest<BannerTargeting>[],
```

Comment:
we now pass `targetingTests` as a parameter so that `bannerSelection.test.ts` doesn't have to trigger the S3 request when running unit tests